sdk-commons/src/main/java/org/openmrs/maven/plugins/utility/DockerHelper.java

```diff
@@ -32,7 +32,7 @@ public class DockerHelper {
     public static final String DOCKER_MYSQL_PASSWORD = "Admin123";
     private static final String DOCKER_HOST_KEY = "dockerHost";
     public static final String DEFAULT_DOCKER_HOST_UNIX_SOCKET = "unix:///var/run/docker.sock";
-    public static final String DEFAULT_HOST_DOCKER_FOR_WINDOWS = "http://127.0.0.1:2375/";
+    public static final String DEFAULT_HOST_DOCKER_FOR_WINDOWS = "tcp://127.0.0.1:2375/";
```

Comment:
We need to be careful with this. It isn't universally true (i.e., the default value here is accurate for some Docker setups, depending on the version of Docker installed). Better would probably be to switch the Docker library to auto-detecting as much as possible.

Reply:
Ok @ibacher .I will look into that. Thanks